pycaret/classification.py

```diff
@@ -927,6 +928,11 @@ def create_model(
         Score grid is not printed when verbose is set to False.
 
 
+    return_train_score: bool, default = False
+        If not False, will evaluate the train value scores.
+        Intended to be fed as an input from the user.
```

Comment:
I would remove this line. Everything is intended to be fed by the user :)

Reply:
Yes, absolutely!